### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by jessicareinh**
Added css Styling to the Form and Edit Function on the my Events Page

### CODE REVIEW

File: components/DeleteButton/index.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+const StyledButton = styled.button`
+  border-radius: 8px;
+  border: 5px;
+  background-color: #bdbdbd;
+  font-family: monospace;
+  color: black;
+  height: 25px;
+  margin: 5px;
+  padding: 3px;
+  cursor: poninter;
```
Comment (Miriam-Voelkl) [inline components/DeleteButton/index.js:12]: typo, should be pointer

File: components/EditEvent/index.js [hunk 2]
```diff
@@ -4,24 +4,60 @@ import styled from "styled-components";
...
-  font-size: 12px;
-  color: #111;
-`;
+const Input = styled.input`
+  font-family: sans-serif;
+  outline: 3px;
+  height: 35px;
+  width: 100%;
+  border: 0px;
+  background-color: whitesmoke;
```
Comment (Miriam-Voelkl) [inline components/EditEvent/index.js:19]: to be consistent, change color to hex

File: components/EventForm/index.js [hunk 3]
```diff
@@ -1,11 +1,64 @@
...
+  outline: 3px;
+  height: 35px;
+  width: 300px;
+  border: 0px;
+  background-color: #d1c4e9;
+  padding: 15px;
+  margin-bottom: 20px;
+
+  &:focus {
+    background-color: whitesmoke;
```
Comment (Miriam-Voelkl) [inline components/EventForm/index.js:27]: as above, you could use the hex code instead of the named color

File: components/MyEventsList/index.js [hunk 4]
```diff
@@ -1,50 +1,104 @@
...
+  border-radius: 8px;
+  border: 5px;
+  box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.7);
+  background-color: #bdbdbd;
+  font-family: monospace;
+  color: black;
+  height: 25px;
+  margin: 15px;
+  padding: 3px;
+  cursor: poninter;
```
Comment (Miriam-Voelkl) [inline components/MyEventsList/index.js:43]: same typo here

**Review by Miriam-Voelkl — COMMENTED** [review]
I like the new styling ✨ I noticed a few typos you can correct and also a couple of named colors you could change to hex colors.
this may be sort of qa more than pr, but I also checked the preview to see the styling in action and noticed that on the add events page, the submit button overlaps with the nav links: 
Screenshot 2024-02-21 at 14 07 21

and if the name of the added event is too long, it runs out of the container:
Screenshot 2024-02-21 at 14 08 13
we fixed the same thing in our notes fields by adding `word-wrap: break-word;`

**Comment by jessicareinh** [comment]
Nice, thanks for your tips almost everything is fixed!

For the overlapping links from your qa: Which device did you use? I can not find this bug, does no matter which device I choose. Also these links will not be necessary anymore. Because Thanh implemented our navbar, so after merge they will be obsolete anyways.

**Comment by somaSeb** [comment]
ChefsKissFrenchChefGIF

**Pull request merged**